client/app/email-domain/email/responder/create/email-domain-email-responder-create.html

```diff
@@ -116,47 +116,39 @@
                             <label class="control-label col-md-4 required" for="date-responder-start"
                                    data-i18n-static="email_tab_modal_create_responder_start"></label>
                             <div class="col-md-6">
-                                <a class="input-group"
-                                   data-date-time-picker
-                                   data-dtp-autoclose="true"
-                                   data-dtp-min-view-mode="days"
-                                   data-dtp-model-timezone="local"
-                                   data-dtp-pick-time="true"
-                                   data-dtp-placement="bottom"
-                                   data-dtp-type="date"
-                                   data-ng-change="ctrl.responderDateStartCheck()"
-                                   data-ng-model="ctrl.model.responderDateStart">
-                                    <input type="text" class="form-control" id="date-responder-start" name="responderDateStart" required>
-                                    <div class="input-group-addon"
-                                          data-uib-tooltip="{{i18n.period_tooltip_from}}"
-                                          data-tooltip-placement="bottom">
+                                <div class="input-group"
+                                     data-moment-picker="ctrl.model.responderDateStart"
+                                     data-change="ctrl.responderDateStartCheck()"
+                                     data-format="YYYY-MM-DD HH:mm"
+                                     data-locale="{{translator.locale}}"
+                                     data-start-view="month">
+                                    <input type="text" class="form-control" id="date-responder-start" name="responderDateStart" required
+                                           data-ng-model="ctrl.model.responderDateStart"
+                                           data-ng-model-options="{ updateOn: 'blur' }">
+                                    <div class="input-group-addon">
```

Comment:
Can we use it with keyboard ? Can't see a button to open the calendar

It's not a blocking issue, we can use directly the input, in this case, we must be careful to specify the format ;)